include/swift/Sema/TypeCheckRequest.h

```diff
@@ -97,6 +98,9 @@ class TypeCheckRequest {
 
 #include "swift/Sema/TypeCheckRequestKinds.def"
     }
+
+    // Work around MSVC warning: not all control paths return a value
```

Comment:
Nit: I'm not sure the comment is really needed. I suppose someone in theory might attempt to remove the llvm_unreachable() and do so if they don't get a warning on a particular platform, but that seems unlikely.